examples/gno.land/r/demo/boards2/post.gno

```diff
@@ -271,6 +282,29 @@ func (post *Post) RenderSummary() string {
 	return s
 }
 
+func (post *Post) renderSourcePost(indent string) (string, *Post) {
+	if post.repostBoardID == 0 {
+		return "", nil
+	}
+
+	// TODO: figure out a way to decouple posts from a global storage.
```

Comment:
```
	// TODO: figure out a way to decouple posts from a global storage.
```

I'm thinking we should eventually decouple it by having the render functions implemented as realm functions instead of `Post` methods, otherwise `repostBoardID` should instead be `repostBoard` being of type `*Board`.

cc @salmad3

Reply:
This would be a separate issue.